website/docs/additional-libraries/boilerplate.md

```diff
@@ -0,0 +1,11 @@
+---
+id: boilerplate
+title: Eightshift Boilerplate
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-boilerplate)
+
+All the tools you need to start building a modern WordPress project using the latest front-end development tools.
```

Comment:
```suggestion
All the tools you need to start building a modern WordPress project using the latest front end development tools.
```
I _do_ hate it with the space in-between, but looks like that's the correct way to write it 🥲